packages/ui/src/app/pages/WorkingGroups/WorkingGroup/WorkingGroup.stories.tsx

```diff
@@ -96,3 +147,148 @@ export default {
+      await userEvent.click(nextButton)
+    })
+    await step('Application Form', async () => {
+      await waitFor(() => expect(nextButton).toBeDisabled())
+      await userEvent.type(modal.getByRole('textbox'), '🐁?')
+      await waitFor(() => expect(nextButton).toBeEnabled())
+      await userEvent.click(modal.getByText('Add new question'))
+      await waitFor(() => expect(nextButton).toBeDisabled())
+      await userEvent.click(modal.getAllByText('Long answer')[1])
+      await userEvent.type(modal.getAllByRole('textbox')[1], '🐘?')
+      await waitFor(() => expect(nextButton).toBeEnabled())
+      await userEvent.click(nextButton)
+    })
+    await step('Staking Policy & Reward', async () => {
+      const createButton = getButtonByText(modal, 'Create Opening')
+      expect(createButton).toBeDisabled()
+      await userEvent.type(modal.getByLabelText('Staking amount *'), '100')
+      await userEvent.clear(modal.getByLabelText('Role cooldown period'))
+      await userEvent.type(modal.getByLabelText('Role cooldown period'), '1000')
+      await userEvent.type(modal.getByLabelText('Reward amount per Block'), '0.1')
+      await waitFor(() => expect(createButton).toBeEnabled())
+      await userEvent.click(createButton)
+    })
+
+    await step('Sign transaction and Create', async () => {
+      expect(await modal.findByText('You intend to create an Opening.'))
+      await userEvent.click(modal.getByText('Sign transaction and Create'))
+    })
+
+    step('Transaction parameters', () => {
+      const [description, openingType, stakePolicy, rewardPerBlock] = args.onCreateOpening.mock.calls.at(-1)
+
+      expect(stakePolicy.toJSON()).toEqual({
+        stakeAmount: 100_0000000000,
+        leavingUnstakingPeriod: 1000,
+      })
+      expect(new BN(rewardPerBlock).toNumber()).toEqual(1000000000)
+
+      expect(openingType).toEqual('Regular')
+      expect(metadataFromBytes(OpeningMetadata, description)).toEqual({ ...WG_OPENING_DATA })
+    })
+  },
+}
+export const CreateOpeningImport: Story = {
+  play: async ({ args, canvasElement, step }) => {
+    const screen = within(canvasElement)
+    const modal = withinModal(canvasElement)
+
+    await userEvent.click(screen.getByText('Add opening'))
+    expect(modal.getByText('Create Opening'))
+    const nextButton = getButtonByText(modal, 'Next step')
+
+    await step('Import File', async () => {
+      const importButton = getButtonByText(modal, 'Import')
+
+      await userEvent.click(importButton)
+
+      const uploadField = await modal.findByText(/Browse for file/)
+      const previewImportButton = getButtonByText(modal, 'Preview Import')
+      expect(previewImportButton).toBeDisabled()
+
+      await userEvent.upload(
+        uploadField,
+        new File([JSON.stringify(WG_JSON_OPENING)], 'file.json', { type: 'application/json' })
+      )
+      expect(await modal.findByText(/File imported successfully, preview your input/))
```

Comment:
It usually doesn't matter but here since there is a file upload that is difficult to reproduce, please end the "Import File" step and start another step, e.g:
```suggestion
    })
    await step('Check imported data', async () => {
      expect(await modal.findByText(/File imported successfully, preview your input/))
```
It's just to be able to got through the design easily with the Interactions tab.